lib/modules/datasource/git-refs/readme.md

```diff
@@ -1,13 +1,15 @@
-This datasource can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+You can use this datasource plus [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to update dependencies that are not supported by Renovate.
+
+The `git-refs` datasource returns a reference from a Git repository.
```

Comment:
> The `git-refs` datasource returns a reference from a Git repository.

Do we mean reference _from_ or reference _to_ here?

Reply:
From

References are a thing in git repos: https://git-scm.com/book/en/v2/Git-Internals-Git-References